imod/schemata.py

```diff
@@ -77,6 +77,18 @@ def scalar_None(obj):
         return (len(obj.shape) == 0) & (~obj.notnull()).all()
 
 
+def align_other_obj_with_coords(obj: GridDataArray, other_obj: GridDataArray) -> Tuple[xr.DataArray, xr.DataArray]:
```

Comment:
this function takes GridDataArrays as input (either xarray or xugrid), but it returns xarray DataArrays. Is that intentional? From the descritpion of it, it should return the same type of objects as it gets in.

Reply:
This is intentional, as no xugrid.aligns exist. It also is not an issue, as no calls to ``.ugrid`` functions are made in the validation and the data is not passed on to do other things after the validation. The type annotation serves as a warning, and it worked as it warned you!